examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -136,8 +127,8 @@ export class TaskList extends DataObject implements ITaskList {
         }
 
         const [nameSharedString, prioritySharedCell] = await Promise.all([
-            taskData.name.get(),
-            taskData.priority.get(),
+            taskData.name?.get(),
+            taskData.priority?.get(),
```

Comment:
These conditionals shouldn't be necessary I hope?

```suggestion
            taskData.name.get(),
            taskData.priority.get(),
```

Reply:
Is it guaranteed that the name and priority will always be defined?

Reply:
Yep!  taskData should be either a full PersistedTask (with both available) or else undefined -- we should never be storing partial tasks.